### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4 - Jaques Colin, Iorio Alexandre, Mulugeta Theodros, Slimani Walid

### DISCUSSION THREAD

**Discussion by SlWa99** [heig-vd-dai-course/heig-vd-dai-course#193] [Show and tell]
We a re going to make a "game" called GreenMix. It's simulated a vegetable-growing simulator !
Stay tuned  👍🌯  !

https://github.com/AlexandreIorio/GreenMix

**Comment by Theodrosrun** [comment]
- [Tag](https://github.com/AlexandreIorio/GreenMix/releases/tag/Greenmix-v1.0.0)
- [Repo](https://github.com/AlexandreIorio/GreenMix)
